controllers/csiscaleoperator_controller.go

```diff
@@ -816,16 +1018,22 @@ func (r *CSIScaleOperatorReconciler) reconcileSecurityContextConstraint(instance
 			newSCC.SetResourceVersion(SCC.GetResourceVersion())
 			err = r.Client.Update(context.TODO(), newSCC)
 			if err != nil {
-				logger.Error(err, "failed to update SecurityContextConstraints.")
+				message := "Failed to update SecurityContextConstraints " + newSCC.GetName()
+				logger.Error(err, message)
+				// TODO: Add event.
+				meta.SetStatusCondition(&crStatus.Conditions, metav1.Condition{
+					Type:    string(config.StatusConditionSuccess),
+					Status:  metav1.ConditionFalse,
+					Reason:  string(csiv1.ResourceUpdateError),
+					Message: message,
+				})
 				return err
 			}
 			logger.Info("SecurityContextConstraint updated successfully.", "Users", csiaccess_users)
 		} else {
 			logger.Info("SecurityContextConstraint is up to date, no updates required.")
 		}
 	}
-
-	logger.Info("exiting reconcileSecurityContextConstraint()")
```

Comment:
Don't remove this log. It can help in debugging.

Reply:
Added back.